charts/pelorus/Chart.yaml

```diff
@@ -14,9 +14,9 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.0.7
+version: 2.0.7-rc.1
```

Comment:
I updated as we discussed in https://github.com/konveyor/pelorus/discussions/811 but chart lint breaks. How should we proceed?

Reply:
we should ignore chart lint at the moment and address this as another PR to cover #811

Reply:
https://github.com/konveyor/pelorus/issues/868

Reply:
ok